docs/pages/database-access/guides/redis.mdx

```diff
@@ -0,0 +1,350 @@
+---
+title: Database Access with Redis
+description: How to configure Teleport Database Access with Redis.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="9.0"
+>
+  Database access for Redis is available starting from Teleport `9.0`.
+</Details>
+
+This guide will help you to:
+
+- Install and configure Teleport.
+- Configure mutual TLS authentication between Teleport and your Redis database.
+- Connect to Redis database through Teleport.
+
+## Prerequisites
+
+- Teleport version `9.0` or newer.
+- Redis database (standalone or cluster) version `6.0` or newer.
+- `redis-cli` installed and added to your system's `PATH` environment variable.
+
+<Admonition type="note" title="Note">
+  Redis `7.0` and RESP3 (REdis Serialization Protocol) are currently not supported.
+</Admonition>
+
+## Step 1/5. Set up Teleport Auth and Proxy
+
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+
+## Step 2/5. Create a Teleport user
+
+(!docs/pages/includes/database-access/create-user.mdx!)
```

Comment:
I think we need to copy the partial into this page and modify it, since there's a lot that doesn't apply to Redis.

Reply:
Which parts do you think should be altered?

Reply:
Looking at this again, saying "a lot" was probably misleading. The part that throws me is that there's an example of updating the user's `traits` to include all `db_names`, followed by an `Admonition` that "The `db_names` property only has effect on PostgreSQL and MongoDB database engines." Is the idea here that if this user were to also access Postgres/Mongo, they could view all database names for those servers?

Reply:
Yes, this trait allows you to connect to all databases and all names/schemas etc. I think it's worth to keep that note, otherwise someone can be suprised by the fact that `db_names` doesn't apply to Redis.